drivers/virtio/net/virtio_net.c

```diff
@@ -150,6 +150,10 @@ struct virtio_net_device {
 	__u8 state;
 	/* RX promiscuous mode. */
 	__u8 promisc : 1;
+	/* Expected amount of descriptors per buffer */
+	__u8 buf_descr_count : 2;
+#define VIRTIO_NET_BUF_DESCR_COUNT_INLINE		1
```

Comment:
Probably subjective to my personal liking :nerd_face: :point_up:. But I find it more conventional that the bitfield def's are above the declaration of the actual field.